source/ProgArgs.cpp

```diff
@@ -2931,6 +2939,7 @@ void ProgArgs::setFromPropertyTreeForService(bpt::ptree& tree)
 	doS3AclVerify = tree.get<bool>(ARG_S3ACLVERIFY_LONG);
 	doS3ListObjVerify = tree.get<bool>(ARG_S3LISTOBJVERIFY_LONG);
 	doStatInline = tree.get<bool>(ARG_STATFILESINLINE_LONG);
+    doS3BucketTaggingInline = tree.get<bool>(ARG_S3STATDIRSINLINE_LONG);
```

Comment:
`doS3BucketTaggingInlineVerify` seems to be missing here. 
in this function, all the variables get packaged to be sent to the service instances for a distributed run. thus, all variables here also need to appear in `ProgArgs::getAsPropertyTreeForService()` through which the service instances receive them.